packages/firestore/src/platform/platform.ts

```diff
@@ -43,6 +43,13 @@ export interface Platform {
   /** Converts a binary string to a Base64 encoded string. */
   btoa(raw: string): string;
 
+  /**
+   * Generates `nBytes` of random bytes.
+   *
+   * If `nBytes <= 0` , an error will be thrown.
```

Comment:
nit: nBytes==0 is valid; please update this comment to state that it will only reject values that are *strictly* less than zero.